ecosystem/vault-lambda-extension/lambda-as_an_archive_caching.tf.disabled

```diff
@@ -5,7 +5,7 @@ resource "aws_lambda_function" "function" {
   filename      = "./demo-function/demo-function.zip"
   handler       = "main"
   runtime       = "provided.al2"
-  layers        = ["arn:aws:lambda:${var.aws_region}:634166935893:layer:vault-lambda-extension:6"]
+  layers        = ["arn:aws:lambda:${var.aws_region}:634166935893:layer:vault-lambda-extension:13"]
 
```

Comment:
I think we need to update `VAULT_ADDR ` and set it to `http://127.0.0.1:8200`
and add `VLE_VAULT_ADDR           = "http://${aws_instance.vault-server.public_ip}:8200",`